builder/dockerfile/parser/testfiles/health/result

```diff
@@ -0,0 +1,9 @@
+(from "debian")
```

Comment:
Disappointing that s-expressions have worked their way in. These should be table based tests but that seems to be out of scope of this review.